frontend/src/modules/icons/Power25Icon.tsx

```diff
@@ -8,6 +14,15 @@ const useStyles = makeStyles((theme: Theme) => ({
   },
 }));
 
+/**
+ * Power25Icon
+ *
+ * @component some description
+ *
+ * @param {SvgIconProps} props - SVG props to override
+ *
+ * @returns {JSX.Element} - some description
```

Comment:
same here

Reply:
Yes , that's right. Updating it in icon files as well